test/apdu-tests/apdu.sh

```diff
@@ -4,9 +4,9 @@ set -Eeuo pipefail
 root="$(git rev-parse --show-toplevel)"
 
 # Uncomment this line if you wish to send apdus to speculos (if you do, make sure to comment out the next line accordingly)
-#nix-shell "$root/nix/ledgerblue.nix" -A shell --pure --run 'LEDGER_PROXY_ADDRESS=127.0.0.1 LEDGER_PROXY_PORT=9999 python -m ledgerblue.runScript' 
+#nix-shell "$root/nix/ledgerblue.nix" -A shell --pure --run 'LEDGER_PROXY_ADDRESS=127.0.0.1 LEDGER_PROXY_PORT=9999 python -m ledgerblue.runScript'
 
-nix-shell "$root/nix/ledgerblue.nix" -A shell --pure --run 'python -m ledgerblue.runScript' 
+nix-shell "$root/nix/ledgerblue.nix" -A shell --pure --run 'python -m ledgerblue.runScript'
```

Comment:
As nix folder is deleted, this script needs to be changed.

Reply:
Probably, let's open on issue for that as this PR is not related to nix

Reply:
https://github.com/trilitech/ledger-app-tezos-baking/issues/55